code/modules/chemistry/Reagents-Transmutation.dm

```diff
@@ -0,0 +1,118 @@
+#define MIN_REAGENT_FOR_CONVERSION 5
+datum
```

Comment:
I realise this is copy pasted from reagent code, but indent pathing is gross and hard to search. Maybe switch to absolute pathing (`/datum/reagent/transmutation`)

Reply:
Done, I'm still getting used to dm and have been following patterns I see, but as with any old codebase I guess there's good and bad patterns. I will do this in the future, the searching problem this causes makes 100% to me, thanks for this feedback.